xml/System/ArraySegment`1.xml

```diff
@@ -1664,10 +1637,11 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Copies the contents of this array segment into a new array.</summary>
+        <returns>An array containing the data in the current array segment.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidOperationException">The underlying array is <see langword="null" />.</exception>
```

Comment:
For all of these "the underlying array is null" cases, that describes the runtime test, but since the public constructors all guard against the array being null, that test is really "when executed on `default(ArraySegment<T>))`".

I can't come up with a "default value" phrasing that feels both natural and novice-friendly, but that concept is a better explanation of where the exception is coming from (in terms of user action/cause)

Reply:
@rpetrusha would you have a suggestion here on how we could phrase this?

Reply:
This is a difficult one -- I don't think that the exception condition can be novice-friendly. How about:

\`default(ArraySegment<T>)\` cannot be converted to an array.

or more generally

This operation cannot be performed on a \`default(ArraySegment\<T>)\`.

Reply:
Thanks Ron! I like the second one better. @bartonjs should I go with that?

Reply:
@mairaw I think the suggested description is good and we should go with that. Stephen approved. Jeremy is at DEFCON so he won't be available to reply.

Reply:
"This operation cannot be performed on a `default(ArraySegment<T>)`." looks fine to me; but if prefixed with the implicit "InvalidOperationException is thrown when" it seems like "cannot" should be "is".  But maybe I'm missing a nuance in the style/tone.